### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by SiqiZheng94**
- Add a frontend template
- Add a page to show all dishes from the menu

### CODE REVIEW

File: frontend/src/assets/logo-header.jpeg
Comment (tejue) [inline frontend/src/assets/logo-header.jpeg:1]: Love it, your personal logo :)

File: frontend/src/pages/Dishes.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                open={modalVisible}
+                onCancel={handleModalCancel}
+                footer={null}
+            >
+                <DishEditForm
+                    dish={editingDish}
+                    onCancel={handleModalCancel}
+                    onFinish={() => {
+                        // Handle form submission and update the dish data
+                        // add the app logic here
```
Comment (tejue) [inline frontend/src/pages/Dishes.tsx:97]: Don't forget to clean your code :)

File: frontend/src/pages/Dishes.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+            form.setFieldsValue(dish);
+        }
+    }, [dish, form]);
+
+    const handleSubmit = () => {
+        form
+            .validateFields()
+            .then((values) => {
+                // Handle form submission and update the dish data
+                // You can add your logic here
```
Comment (tejue) [inline frontend/src/pages/Dishes.tsx:120]: Dont forget to clean your code

File: frontend/src/pages/Inventory.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {Typography} from "antd";
+
+function Inventory () {
+    return (
+        <div>
```
Comment (tejue) [inline frontend/src/pages/Inventory.tsx:5]: You might not need these extra div's

File: frontend/src/pages/Orders.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import {Typography} from "antd";
+
+function Orders () {
+    return (
+        <div>
```
Comment (tejue) [inline frontend/src/pages/Orders.tsx:5]: Same here. you might not need the div's (except this Typo-thingy makes it necessary)

File: frontend/src/vite-env.d.ts
Comment (tejue) [inline frontend/src/vite-env.d.ts:1]: mmhh, is this the right file? I thought this should be in the vite.config.ts

**Review by tejue — APPROVED** [review]
lgtm :) 
I'm not so familiar with these extra libraries for styling as well as forms but besides that I left you some comments.

File: frontend/src/vite-env.d.ts
Comment (SiqiZheng94) [reply frontend/src/vite-env.d.ts:1]: You're right, very attentive!

**Pull request merged**